src/pages/_id/index.vue

```diff
@@ -245,6 +248,19 @@ export default {
     userAvatar() {
       return this.userInfo && this.userInfo.avatar;
     },
+    sortedOwnedNFTClassId() {
+      const sorted = [...this.ownedNFTClassId];
+      sorted.sort((a, b) => {
+        const aIsWritingNFT = isWritingNFT(this.getNFTClassMetadataById(a))
+          ? 1
+          : 0;
+        const bIsWritingNFT = isWritingNFT(this.getNFTClassMetadataById(b))
+          ? 1
+          : 0;
+        return bIsWritingNFT - aIsWritingNFT;
```

Comment:
Would be even clearer if we can just return 0, 1, -1 according to a/b's true false value